doc/examples/mr-example/pom.xml

```diff
@@ -61,17 +61,17 @@
     <dependency>
       <groupId>org.apache.avro</groupId>
       <artifactId>avro</artifactId>
-      <version>1.7.5</version>
+      <version>1.9.0</version>
```

Comment:
there's a reference up above to the avro-maven-plugin that needs to be updated to match the other references here.

https://github.com/apache/avro/pull/238/files#diff-e1df2247e8ac37938fc53089e90c9dd1R43